docs/source/conf.py

```diff
@@ -85,11 +97,108 @@
 
 katex_prerender = True
 
+# General information about the project.
+project = "PyTorch"
+copyright = "PyTorch Contributors"
+author = "PyTorch Contributors"
+torch_version = str(torch.__version__)
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+# TODO: change to [:2] at v1.0
+version = "main (" + torch_version + " )"
+# The full version, including alpha/beta/rc tags.
+# TODO: verify this works as expected
```

Comment:
Does it?

Reply:
this is just moved from below. We need version and release variables before html_theme_options and html_context

Reply:
It does now: https://docs-preview.pytorch.org/pytorch/pytorch/149331/index.html